dependencies.yaml

```diff
@@ -260,9 +260,6 @@ dependencies:
           - pytest
           - pytest-cov
           - pytest-xdist
-      - output_types: conda
-        packages:
-          - panel>=1.0,<=1.3.4
```

Comment:
Interested to see how this goes

If we need to ignore a specific version, maybe we can do `!=`

Though if this is just the case of working around a past solver issue or limiting to a set of tested releases, then simply dropping seems reasonable

Maybe we can follow up with the cuxfilter team on which makes more sense

Reply:
Yeah, AFAIK, there are no issues at runtime, so as long as the tests pass (seems like they do), this should be fine